### DISCUSSION TITLE
I would like to export data on the forces that a rigid body receives from a fluid or another rigid body.

### DISCUSSION THREAD

**Discussion by fluidkocy** [InteractiveComputerGraphics/SPlisHSPlasH#303] [Q&A]
I want to export the force data that a rigid body receives from a fluid or another rigid body.

How can I rewrite the scene file?
To export pressure, density, and velocity data for water, write "particleAttributes": "pressure;velocity;density" in the configuration section of the scene file, and the vtk file will include the data. 
In the same way, I want to write out as a vtk file the data of the force received from a rigid body fluid.

**Comment by janbender** [comment]
This is not implemented yet. You will have to extend the exporter or write an own one. 

Rigid-rigid forces are handled in the PBD library, so you cannot export them from SPlisHSPlasH.
